components/HelpScreen.tsx

```diff
@@ -1,12 +1,77 @@
 import closeIcon from "./images/closeIcon.png";
-import statusLegend from "./images/helpLegend.svg";
+import greenIcon from "./images/greenIcon.svg";
+import yellowIcon from "./images/yellowIcon.svg";
+import redIcon from "./images/redIcon.svg";
 import styles from "./HelpScreen.module.css";
 import Image from "next/image";
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
 
 export type HelpScreenProps = {
   closeHelp: (_value: boolean | ((_prev: boolean) => boolean)) => void;
 };
 
+// class created to make linking of descriptions to config easier later on
+class VersionDefinition {
+  description: string;
+
+  constructor(description: string) {
+    this.description = description;
+  }
+}
+
+// defines red, yellow and green traffic light descriptions
+let redDef = new VersionDefinition("Current major version behind by > 1 major OR > 6 minors.");
+let yellowDef = new VersionDefinition("Current minor version behind by 5 or 6 minors.");
+let greenDef = new VersionDefinition("Current minor version behind by < 5 minors.");
```

Comment:
Everything looks great, I'd just change the symbols `>` and `<` with the words `more` and `less` just for readability reasons. At least it makes it a little more hard for me to read.